WickedEngine/wiRawInput.cpp

```diff
@@ -1,7 +1,21 @@
 #include "wiRawInput.h"
 #include "wiPlatform.h"
 
-#if defined(_WIN32) && !defined(PLATFORM_UWP)
+#ifdef _WIN32
```

Comment:
For platforms where there is no Raw input API, we don't care so all these changes should be removed. Raw input is only Windows-desktop specific API as far as I am aware.

Reply:
Alright. I'll #ifdef it out of the Linux build